backend/langate/network/models.py

```diff
@@ -161,7 +168,7 @@ def delete_user_device(Device):
         return DeviceManager.delete_device(Device.mac)
 
     @staticmethod
-    def edit_device(device: Device, mac, name, mark=None):
+    def edit_device(device: Device, mac=None, name=None, mark=None, bypass=None):
```

Comment:
Same as last comment, None makes no sense as default value it should be False.

Reply:
I think it does here since it should default to the previous value and not False (right?)